examples/regression/1-quickstart/plot_homoscedastic_1d_data.py

```diff
@@ -125,29 +126,58 @@ def plot_1d_data(
         ("linear", LinearRegression(fit_intercept=False)),
     ]
 )
+polyn_model_quant = Pipeline(
+    [
+        ("poly", PolynomialFeatures(degree=4)),
+        ("linear", QuantileRegressor(
+            alpha=1e-9,
+            fit_intercept=False,
+        )),
+    ]
+)
 
-Params = TypedDict("Params", {"method": str, "cv": int})
 STRATEGIES = {
-    "jackknife": Params(method="base", cv=-1),
-    "jackknife_plus": Params(method="plus", cv=-1),
-    "jackknife_minmax": Params(method="minmax", cv=-1),
-    "cv": Params(method="base", cv=10),
-    "cv_plus": Params(method="plus", cv=10),
-    "cv_minmax": Params(method="minmax", cv=10),
+    "jackknife": {"method": "base", "cv": -1},
+    "jackknife_plus": {"method": "plus", "cv": -1},
+    "jackknife_minmax": {"method": "minmax", "cv": -1},
+    # "cv": {"method": "base", "cv": 10},
+    "cv_plus": {"method": "plus", "cv": 10},
+    "cv_minmax": {"method": "minmax", "cv": 10},
+    "quantile": {"method": "quantile", "cv": "split"},
 }
 fig, ((ax1, ax2, ax3), (ax4, ax5, ax6)) = plt.subplots(
     2, 3, figsize=(3 * 6, 12)
 )
 axs = [ax1, ax2, ax3, ax4, ax5, ax6]
 for i, (strategy, params) in enumerate(STRATEGIES.items()):
-    mapie = MapieRegressor(
-        polyn_model, agg_function="median", n_jobs=-1, **params
-    )
-    mapie.fit(X_train.reshape(-1, 1), y_train)
-    y_pred, y_pis = mapie.predict(
-        X_test.reshape(-1, 1),
-        alpha=0.05,
-    )
+    if strategy == "quantile":
+        mapie = MapieQuantileRegressor(
+            polyn_model_quant, **params
+        )  # type: ignore
+        X_train, X_calib, y_train, y_calib = train_test_split(
+            X_train,
+            y_train,
+            test_size=0.5,
+            random_state=1
+        )
+        mapie.fit(
+            X_train.reshape(-1, 1),
+            y_train,
+            X_calib.reshape(-1, 1),
+            y_calib
+        )
+        y_pred, y_pis = mapie.predict(
+            X_test.reshape(-1, 1)
+        )
+    else:
+        mapie = MapieRegressor(
+            polyn_model, agg_function="median", n_jobs=-1, **params
+        )  # type: ignore
```

Comment:
Is is still needed ?

Reply:
Same as for all the others, if we don't put this when making `make type-check`, we get this error message:
"examples/regression/1-quickstart/plot_heteroscedastic_1d_data.py:158: error: Argument after ** must be a mapping, not "object"". This is also what has made the last PR fail for some tests on the Ubuntu machine.